packages/mobile/src/send/SendAmount.tsx

```diff
@@ -206,6 +215,21 @@ export class SendAmount extends React.Component<Props, State> {
     }
   }
 
+  isPaymentLimitReached = (now: number) => {
+    // we are only interested in the last 24 hours
+    const paymentsLast24Hours = this.props.recentPayments.filter(
+      (p: PaymentInfo) => timeDeltaInHours(now, p.timestamp) < 24
+    )
+
+    const initial = new BigNumber(this.state.amount)
```

Comment:
Use `parseInputAmount` here, see elsewhere in this file for examples

Reply:
I switched to using the existing method this.getDollarsAmount() and then calling toNumber() on it. I will ping Jean as well to make sure this is the right method to use.

Reply:
@jeanregisser could you please take a look at this PR, in particular to the local currency conversions?